ci-operator/step-registry/openshift/upgrade/qe/sanity/arch-migration/upi/openshift-upgrade-qe-sanity-arch-migration-upi-chain.yaml

```diff
@@ -0,0 +1,11 @@
+chain:
+  as: openshift-upgrade-qe-sanity-arch-migration-upi
```

Comment:
Sorry, I didn't find where is this chain used in the PR. Could you explain a little bit ?

Reply:
Right. I just copied the whole folder - `openshift/upgrade/qe/test/arch-migration` to create this sanity folder, so the `upi` folder is copied together. I guess the upi chain is not referenced for now, maybe before we need it on some specific profiles. I think it is harmless to keep it, if in the future, we select some UPI profiles to run testing, we can easily add the ci jobs to by calling this chain.

Reply:
I'd prefer drop it here.  We can add it in the future if we need it.

Reply:
Done.